### ISSUE TITLE
vue/html-indent not allowing conditions with aligned indentation

### ISSUE THREAD

**Issue by chrisvfritz** [vuejs/eslint-plugin-vue#264]
**Tell us about your environment**

* **ESLint Version:** 4.12.0
* **eslint-plugin-vue Version:** 4.0.0-beta.0
* **Node Version:** 8.9.1

**Please show your full configuration:**

```
module.exports = {
  root: true,
  parserOptions: {
    parser: 'babel-eslint',
    sourceType: 'module'
  },
  extends: ['standard', 'plugin:vue/recommended']
}
```

**What did you do? Please include the actual source code causing the issue.**

```vue
<template>
  <div
    :class="{
      foo: (
        a === b &&
        c === d
      )
    }"
  />
</template>
```

**What did you expect to happen?**

I expected this to be an acceptable style.

**What actually happened? Please include the actual, raw output from ESLint.**

```
  ✘  vue/html-indent  Expected indentation of 10 spaces but found 8 spaces
  src/components/component.vue:6:1
          c === d
   ^
```
